From d95e7b1f98332e3d6d4d3f41e496aa758facb3ee Mon Sep 17 00:00:00 2001
From: Ranjani Vaidyanathan <ra5478@freescale.com>
Date: Thu, 13 Oct 2011 13:22:24 -0500
Subject: ENGR00159959: MX6-Updated CPU voltages for different frequencies

Update CPU voltages for different frequencies based on characterized values.

Signed-off-by: Ranjani Vaidyanathan <ra5478@freescale.com>
---
 arch/arm/mach-mx6/cpu_op-mx6.c |    8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm/mach-mx6/cpu_op-mx6.c b/arch/arm/mach-mx6/cpu_op-mx6.c
index 5b03abb..ae3bff9 100644
--- a/arch/arm/mach-mx6/cpu_op-mx6.c
+++ b/arch/arm/mach-mx6/cpu_op-mx6.c
@@ -29,7 +29,7 @@ static struct cpu_op mx6_cpu_op[] = {
 	 .mfd = 11,
 	 .mfn = 5,
 	 .cpu_podf = 0,
-	 .cpu_voltage = 1100000,},
+	 .cpu_voltage = 1150000,},
 	{
 	 .pll_rate = 792000000,
 	 .cpu_rate = 792000000,
@@ -38,17 +38,17 @@ static struct cpu_op mx6_cpu_op[] = {
 	 .mfd = 2,
 	 .mfn = 1,
 	 .cpu_podf = 0,
-	 .cpu_voltage = 1025000,},
+	 .cpu_voltage = 1150000,},
 	 {
 	  .pll_rate = 792000000,
 	  .cpu_rate = 400000000,
 	  .cpu_podf = 1,
-	  .cpu_voltage = 1025000,},
+	  .cpu_voltage = 1050000,},
 	{
 	 .pll_rate = 792000000,
 	 .cpu_rate = 167000000,
 	 .cpu_podf = 4,
-	 .cpu_voltage = 1000000,},
+	 .cpu_voltage = 900000,},
 };
 
 struct cpu_op *mx6_get_cpu_op(int *op)
-- 
1.7.9.5

